README.md

```diff
@@ -299,8 +301,8 @@ class fourthcoffee(
 }
 ~~~
 
-As you can see, you can mix and match dsc resources with common puppet resources.
-All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
+As you can see, you can mix and match dsc resources with common Puppet resources.
```

Comment:
Anywhere we find the word DSC, unless it is based on the module name which is `puppetlabs-dsc`, should probably be capitalized. We didn't do good with that distinction in the readme at all :/